src/main/java/org/apache/commons/lang3/SerializationUtils.java

```diff
@@ -122,7 +122,7 @@ public SerializationUtils() {
      * @since 3.3
      */
     public static <T extends Serializable> T roundtrip(final T msg) {
-        return (T) SerializationUtils.deserialize(SerializationUtils.serialize(msg));
+        return SerializationUtils.deserialize(SerializationUtils.serialize(msg));
```

Comment:
In the future I'll be more careful when rebasing after splitting changes into separate branches.  The ancient JDK on my other computer thanks you.  :)